crates/gpui/src/platform/linux/window.rs

```diff
@@ -428,4 +428,8 @@ impl PlatformWindow for LinuxWindow {
         let inner = self.0.inner.lock();
         inner.renderer.atlas().clone()
     }
+
+    fn set_graphics_profiler_enabled(&self, enabled: bool) {
+        todo!("linux")
```

Comment:
@kvark @mikayla-maki @ForLoveOfCats: stubbing this out right now on Linux, but I think it would be cool to have something similar in there.

Reply:
Never used `MetalHudEnabled` before, interesting stuff! Not sure if we'd be able to support it transparently on Linux - it's not the graphics driver responsibility to provide a HUD. Maybe there is a Vulkan layer out there that provides similar functionality, which we'll enable. We'll see!

Reply:
from a quick search apparently there is ``VK_LAYER_MESA_overlay`` for this but have not tried it out, if worse comes to worse maybe we suggest people use an external overlay like mangohud or something

Reply:
I can confirm doing ``VK_INSTANCE_LAYERS=VK_LAYER_MESA_overlay cargo r`` does put a little fps overlay thing

Reply:
sweet!